cycamore_input/predicting_the_past.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" standalone="no" ?>
+
+<simulation xmlns:xi="http://www.w3.org/2001/XInclude">
```

Comment:
1. The code runs without issues. From the w3.org website, the xmls:xi stuff seemed to be added in the root tag. I'll try if this run when placed in Line 1.

2. Yes, since in_commod and out_commod recipes is identical for all reactors, we've disabled xml inclusion. Thank you for pointing that out. I'll remove the xmlns:xi stuff.